data/lang/core/en.json

```
@@ -599,10 +599,22 @@
     "description": "",
     "message": "hydrogen atmosphere"
   },
+  "DWARF_PLANET_ICE": {
+    "description": "",
+    "message": "Dwarf planet, ice"
```

This is slightly awkward. Perhaps a better label might be:
```suggestion
    "message": "Dwarf planet, mostly covered in ice"
```

---

Yeah, that doesn't look good. I was trying to get the more specific terms, 'Terrestrial Dwarf' and 'Ice Dwarf' into the game. Only, I didn't think they fit as a first string and also, I guessed that they aren't commonly known. Looking into it but your suggestion is good.